public/app/features/auth-config/fields.tsx

```diff
@@ -448,6 +457,22 @@ export function fieldMap(provider: string): Record<string, FieldData> {
       description: 'Prevent synchronizing users’ organization roles from your IdP.',
       type: 'switch',
     },
+    orgMapping: {
+      label: 'Organization mapping',
+      description: orgMappingDescription(provider),
+      type: 'select',
+      hidden: !contextSrv.isGrafanaAdmin,
```

Comment:
If this field is hidden then its current value will be sent to the API on submit? I think we want to pass the existing value for this field in case the user is not a Grafana admin.

Reply:
Yes, this is a similar problem we have with AllowAssignGrafanaAdmin and wanted to discuss with the team!

Reply:
Done